cmd/changefeed.toml

```diff
@@ -1,6 +1,7 @@
-ignore-txn-commit-ts = []
 
 filter-case-sensitive = false
+ignore-txn-commit-ts = []
+ddl-white-list = [100, 101]
```

Comment:
seems `100` and `101` are invalid DDL ActionType, recommend to use valid value. We should also update the document and put a URL here to tell user what value they can use

Reply:
updated, and I think we should hide `ddl-white-list` to user